### PR TITLE
[XGUARD-4497] displaying list supervisors

### PR DESCRIPTION
**Pull request by SiamakSamie**
### Description
Retrieving list of all supervisors and necessary data

### What kind of change does this PR introduce?

- [x] New feature (adds functionality)
- [x] Bug fix (fixes an issue)
- [x] Refactor (code changes that do **NOT** change functionality)
- [x] Code Coverage (adds Unit Tests)

### Checklist:
Please check if your PR fulfills the following requirements:

- [x] Unit Tests for the changes have been added.
- [x] Contains **NO** breaking changes.
- [x] Associated with a JIRA issue.

### CODE REVIEW

File: src/Models/SupervisorShift.php [hunk 1]
```diff
@@ -55,4 +56,12 @@ public function jobSiteVisits(): HasMany
     {
         return $this->hasMany(JobSiteVisit::class);
     }
+
+    public function getIsActiveAttribute(): bool
+    {
+        if (!$this->end_time && $this->start_time &&  $this->start_time >= Carbon::now()->subDay()) {
```
Comment (ssardinh) [inline src/Models/SupervisorShift.php]: Avoid if true returne true, else return false, just return the expression

File: src/resources/js/components/dashboard/dashboardComponents/UserCard.vue [hunk 2]
```diff
@@ -19,12 +23,19 @@ export default {
...
             type: Object,
             default: () => ({})
 
         }
-    }
+    },
+
+    computed: {
+        isActive() {
+            return this.supervisor.supervisorShifts.some((e) => e.isActive === true);
```
Comment (ssardinh) [inline src/resources/js/components/dashboard/dashboardComponents/UserCard.vue]: What is the point of adding === true ?

File: src/routes/web.php [hunk 3]
```diff
@@ -19,10 +19,13 @@
                 // Coordinator App Data
                 Route::get('/get-role-and-coordinator-id', 'AppController@getRoleAndCoordinatorId');
                 Route::get('/get-admin-page-data', 'AppController@getAdminPageData');
-                Route::get('/get-dashboard-data', 'AppController@getAdminPageData');
                 Route::get('/get-coordinator-profile', 'AppController@getCoordinatorProfile');
                 Route::get('/get-footer-info', 'AppController@getFooterInfo');
 
+                // Supervisors
+
```
Comment (ssardinh) [inline src/routes/web.php]: Remove white space

**Review by ssardinh — CHANGES_REQUESTED** [review]
Please see comments

File: src/Models/SupervisorShift.php (hunk 1 above)
Comment (SiamakSamie) [reply src/Models/SupervisorShift.php]: done

File: src/resources/js/components/dashboard/dashboardComponents/UserCard.vue (hunk 2 above)
Comment (SiamakSamie) [reply src/resources/js/components/dashboard/dashboardComponents/UserCard.vue]: no point at all just reflex

File: src/routes/web.php (hunk 3 above)
Comment (SiamakSamie) [reply src/routes/web.php]: oki dokie

**Review by ssardinh — APPROVED** [review]

**Pull request merged**